gamestonk_terminal/main_helper.py

```diff
@@ -147,7 +147,7 @@ def load(l_args, s_ticker, s_start, s_interval, df_stock):
         "-s",
         "--start",
         type=valid_date,
-        default="2015-01-01",
+        default="2019-01-01",
```

Comment:
I don't think this was supposed, right?

Reply:
As we chatted about, I think 2 years of data by default is fine.